chillwave.js

```diff
@@ -1,14 +1,12 @@
-function makeChillWaveString(string) {
-    let chillWaveString = '';
-
-    for (let i = 0; i < string.length; i++) {
-        let char = string[i];
-        if (char === ' ') {
-            chillWaveString += '  ';
-        } else {
-            chillWaveString += char + ' ';
-        }
-    }
-
-    return chillWaveString;
-}
\ No newline at end of file
+const convert = (str) => str.split('').join(' ');
```

Comment:
Can't `(str)` be `str`?

Reply:
It can be, lad! It all depends on what you prefer.

Reply:
![image](https://user-images.githubusercontent.com/9219224/67981928-467b1780-fbe7-11e9-897b-5c5e995a5eae.png)